.github/workflows/OCV-PR-4.x-RISCV.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
         - /mnt/cache/ci_cache/opencv:/home/ci/.ccache
         - /mnt/cache/binaries_cache:/home/ci/binaries_cache
         - /mnt/cache/dnn-models:/home/ci/dnn-models
+      options: '--user root'
```

Comment:
Looks strange and unsafe,

Reply:
It's OK, we use rootless docker install and it maps root user (0) in container to regular user on machine (e.g. 1001). I've moved this option to the image.